e3sm_diags/plot/cartopy/polar_plot.py

```diff
@@ -1,278 +0,0 @@
-from __future__ import print_function
-
-import os
-
-import cartopy.crs as ccrs
-import matplotlib
-import matplotlib.pyplot as plt
-import numpy as np
-import numpy.ma as ma
-
-from e3sm_diags.driver.utils.general import get_output_dir
-from e3sm_diags.logger import custom_logger
-from e3sm_diags.plot import get_colormap
-
-matplotlib.use("Agg")
-import matplotlib.colors as colors  # isort:skip  # noqa: E402
-import matplotlib.path as mpath  # isort:skip  # noqa: E402
-
-logger = custom_logger(__name__)
-
-plotTitle = {"fontsize": 11.5}
-plotSideTitle = {"fontsize": 9.0}
-
-# Position and sizes of subplot axes in page coordinates (0 to 1)
-panel = [
-    (0.27, 0.65, 0.3235, 0.25),
-    (0.27, 0.35, 0.3235, 0.25),
-    (0.27, 0.05, 0.3235, 0.25),
-]
-
-# Border padding relative to subplot axes for saving individual panels
-# (left, bottom, right, top) in page coordinates
-border = (-0.02, -0.01, 0.14, 0.04)
-
-
-def add_cyclic(var):
-    lon = var.getLongitude()
-    return var(longitude=(lon[0], lon[0] + 360.0, "coe"))
-
-
-def get_ax_size(fig, ax):
-    bbox = ax.get_window_extent().transformed(fig.dpi_scale_trans.inverted())
-    width, height = bbox.width, bbox.height
-    width *= fig.dpi
-    height *= fig.dpi
-    return width, height
-
-
-def plot_panel(n, fig, proj, pole, var, clevels, cmap, title, parameters, stats=None):
-    var = add_cyclic(var)
-    lon = var.getLongitude()
-    lat = var.getLatitude()
-    var = ma.squeeze(var.asma())
-
-    # Contour levels
-    levels = None
-    norm = None
-    if len(clevels) > 0:
-        levels = [-1.0e8] + clevels + [1.0e8]
-        norm = colors.BoundaryNorm(boundaries=levels, ncolors=256)
-
-    # Contour plot
-    ax = fig.add_axes(panel[n], projection=proj)
-    ax.set_global()
-
-    ax.gridlines()
-    if pole == "N":
-        ax.set_extent([-180, 180, 50, 90], crs=ccrs.PlateCarree())
-    elif pole == "S":
-        ax.set_extent([-180, 180, -55, -90], crs=ccrs.PlateCarree())
-
-    cmap = get_colormap(cmap, parameters)
-
-    theta = np.linspace(0, 2 * np.pi, 100)
-    center, radius = [0.5, 0.5], 0.5
-    verts = np.vstack([np.sin(theta), np.cos(theta)]).T
-    circle = mpath.Path(verts * radius + center)
-    ax.set_boundary(circle, transform=ax.transAxes)
-
-    p1 = ax.contourf(
-        lon,
-        lat,
-        var,
-        transform=ccrs.PlateCarree(),
-        norm=norm,
-        levels=levels,
-        cmap=cmap,
-        extend="both",
-    )
-    ax.set_aspect("auto")
-    ax.coastlines(lw=0.3)
-
-    # Plot titles
-    if title[0] is not None:
-        ax.set_title(title[0], loc="left", fontdict=plotSideTitle)
-    if title[1] is not None:
-        ax.set_title(title[1], fontdict=plotTitle)
-    if title[2] is not None:
-        ax.set_title(title[2], loc="right", fontdict=plotSideTitle)
-
-    # Color bar
-    cbax = fig.add_axes((panel[n][0] + 0.35, panel[n][1] + 0.0354, 0.0326, 0.1792))
-    cbar = fig.colorbar(p1, cax=cbax)
-    w, h = get_ax_size(fig, cbax)
```

Comment:
Doesn't appear to have a correspondence. But also doesn't appear to be used.

Reply:
`_add_color_map` covers the first two lines. `w` and `h` are not referenced in various plotter files and was not included in `_add_color_map`.

https://github.com/E3SM-Project/e3sm_diags/blob/7bc413081def922b7b3f8fbf5f747f560f740dcb/e3sm_diags/plot/utils.py#L226-L233